MobModule/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>mob-system</artifactId>
+        <groupId>eu.cloudnetservice</groupId>
```

Comment:
When we are already pedantic about naming, we should make sure to not restrict ourselves to just the base group id.
Maybe something like `eu.cloudnetservice.v2.mobs` would be appropriate